### ISSUE TITLE
Inscriptions not getting sent to mempool

### ISSUE THREAD

**Issue by chipaaas** [ordinals/ord#1626]
Issue has been plaguing me while inscribing. Not sure what causes it but on some inscriptions iIsend, the txns ever get broadcasted to the mempool. Have lots of funds just locked up because of this :/

**Comment by raphjaph** [comment]
When you do `ord wallet transactions`, do you see the transactions that haven't been relayed to the mempool?

**Comment by chipaaas** [comment]
Yes it gives me the txn hash but confirmations = 0. When I looked for the specific txn hash in the mempool, it returns as it not been sent yet.

**Comment by raphjaph** [comment]
How are you looking for the transaction in the mempool?

**Comment by chipaaas** [comment]
I have. It says transaction not found. Here's an example of one of these txns. I'm running on windows 

https://mempool.space/tx/bcfff2bdef3d41c5b8a5abaf48e45b7f3936822018c58bde2fea8ace38281e1a

**Comment by raphjaph** [comment]
The mempool is currently quite full, so if you are paying a very low fee rate other nodes won't let your transaction into the mempool. At the moment there is not much you can do until either the mempool clears or we implement RBF. You can probably also try to cancel the transactions with `bitcoin-cli` but I would not recommend that because you might lose your already made inscriptions.

**Comment by chipaaas** [comment]
The problem is that I've not been using any low fees. For all the stuck txns, I used a fee rate between 15-20

**Comment by raphjaph** [comment]
Can you find the transactions in your local mempool?

`bitcoin-cli getrawmempool | grep <TXID>`

**Comment by Vardation** [comment]
I know this is old AF, but I'm in the same boat (but linux). Fees have been in the 50s for hours and I set mine to 65 about 4 hours ago. It does show the tx in that local mempool when I tried that command you suggested. What does that indicate?

**Comment by raphjaph** [comment]
This could mean that your local node is not sending out your transaction for some reason. Try restarting `bitcoind`. You can also check if you have peers that are connected to you by doing `bitcoin-cli getpeerinfo`

**Comment by Vardation** [comment]
Thanks heaps mate, I'll look in that direction. Cheers for taking the time.
